packages/algob/src/internal/util/scripts-runner.ts

```diff
@@ -18,10 +18,44 @@ async function loadScript (relativeScriptPath: string): Promise<any> {
   }
 }
 
+export async function runDeployASA (
+  relativeYamlPath: string,
+  runtimeEnv: AlgobRuntimeEnv,
+  deployer: AlgobDeployer): Promise<void> {
+}
+
+function displayErr (error: Error | BuilderError | any, relativeScriptPath: string): void {
+  if (error instanceof BuilderError) {
+    throw error;
+  }
+  // TX execution error
+  if (error?.response?.error !== undefined) {
+    throw new BuilderError(
+      ERRORS.BUILTIN_TASKS.SCRIPT_CHAIN_IO_ERROR, {
+        script: relativeScriptPath,
+        errorType: error.response.clientError
+          ? "Client"
+          : "Server",
+        message: error.response?.body?.message
+          ? error.response.body.message
+          : error.response.error.message
+      },
+      error.response.error
+    );
+  }
+  throw new BuilderError(
+    ERRORS.BUILTIN_TASKS.SCRIPT_EXECUTION_ERROR, {
+      script: relativeScriptPath,
+      message: error.message
+    },
+    error
+  );
+}
```

Comment:
Please remove this and use the `parseAlgorandError` function.

Reply:
changed